demo/screens/LandClaim.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
-import { ListView, Image, StyleSheet, View, Text, Button } from "react-native";
+import { StyleSheet, View, Button } from "react-native";
 import Land from "../components/presentational/Land";
+import Colors from "../constants/Colors";
```

Comment:
This is a good candidate for doing the package.json name and then getting it via `@` so we don't need to remember how deep we are in subdirs

Reply:
(That doesn't need to change in this PR, though)

Reply:
Ok